src/tools/rbd/Utils.h

```diff
@@ -82,6 +82,14 @@ struct ProgressContext : public librbd::ProgressContext {
   void fail();
 };
 
+struct ImageCacheState {
+  bool present = false;
+  bool clean;
```

Comment:
Nit: default init all primitive types (and move to `Status.cc`)

Reply:
Done